### ISSUE TITLE
EM101 in Django REST framework

### ISSUE THREAD

**Issue by spaceby** [astral-sh/ruff#13526]
I used the EM101 rule in a DRF project and found some nuances

```python
from rest_framework import serializers


class Example_1(serializers.Serializer):
    def validate(self, attrs):
        # working TRY003 and EM101
        raise serializers.ValidationError('Not found')
```
https://play.ruff.rs/4ec7c931-8782-424a-96ca-6aee8cd51973

```python
from django.utils.translation import gettext_lazy as _

from rest_framework import serializers


class Example_1(serializers.Serializer):
    def validate(self, attrs):
        # nothing
        raise serializers.ValidationError(_('Not found'))  # Error with translation
```
https://play.ruff.rs/3379f37d-c838-4d7a-ae67-d22ddfed06b8

```python
from rest_framework import serializers


class Example_1(serializers.Serializer):
    def validate(self, attrs):
        # nothing
        raise serializers.ValidationError({'field': 'Not found'})  # Error for serializer field
```
https://play.ruff.rs/29d82491-3fad-4af1-8522-f04bafe57bc5

```python
from rest_framework import serializers


class Example_1(serializers.Serializer):
    def validate(self, attrs):
        # working EM101 but shouldn't
        raise self.fail('required')
        # "raise self.fail('required')" this code is not correct, although it works
        # should be just "self.fail('required')"
```
https://play.ruff.rs/f8bc7092-15e6-4328-999c-30ad3ff0813d
